dozer-ingestion/src/connectors/postgres/helper.rs

```diff
@@ -74,7 +74,11 @@ pub fn postgres_type_to_field(
                     .unwrap();
                     Ok(Field::from(date))
                 }
-                Type::JSONB | Type::JSON => Ok(Field::Bson(v.to_vec())),
+                Type::JSONB | Type::JSON => {
```

Comment:
Do we support `Type::JSON`? In the type conversion I only see `Type::JSONB`.

Reply:
we do have support for `Type::JSON`, `Type::JSONB`, `Type::JSON_ARRAY`, `Type::JSONB_ARRAY`

Reply:
In `postgres_type_to_dozer_type` I only see `Type::JSONB`

Reply:
here it is :) - https://github.com/getdozer/dozer/pull/1200/files#diff-23b2d6cb5efb3cee96f64fc2eb5b987363d9099ea56686f03935d41d1ebddb1aR109